spec/page_searcher_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'app_config'
+
+require_relative '../lib/page_searcher'
+require_relative '../lib/root'
+require_relative '../lib/app_config'
+require 'pp'
+
+
+describe PageSearcher do
+
+  describe '#pages' do
+    it 'returns an array of the top four pages for each format' do
+      config = AppConfig.new(File.dirname(__FILE__) + '/test_config.yml')
+      searcher = PageSearcher.new(config, %w{ speech person } )
+      expect(searcher).to receive(:result_set_for_format).with('speech').and_return(speeches)
+      expect(searcher).to receive(:result_set_for_format).with('person').and_return(people)
+      searcher.run
+      expect(searcher.pages).to eq expected_pages
+    end
+  end
+end
+
+def expected_pages
```

Comment:
These methods are globally accessible. It might be better to move them into the `describe`.